src/pytorch_lightning/strategies/fully_sharded_native.py

```diff
@@ -273,6 +309,24 @@ def reduce(
             tensor = sync_ddp_if_available(tensor, group, reduce_op=reduce_op)
         return tensor
 
+    def training_step(self, *args: Any, **kwargs: Any) -> STEP_OUTPUT:
+        # we don't need precision context since casting is done by FSDP
+        # read `mixed_precision``` docstring here: https://pytorch.org/docs/stable/fsdp.html
```

Comment:
```suggestion
        # read `mixed_precision` docstring here: https://pytorch.org/docs/stable/fsdp.html
```

Also, you can :paramref: the argument of the FSDP class in PyTorch docs

Reply:
it's a comment. Will that be useful?